playground/src/components/state-machines-list/state-machines-list.spec.js

```diff
@@ -25,16 +25,22 @@ const createStateMachinesSdkMock = () => ({
   },
 });
 
+const renderApp = (options = {}) => {
```

Comment:
Careful because `renderApp` is exported from `test-utils`

Reply:
Not on this branch yet. After I rebase, yes.